src/main/java/org/springframework/integration/kafka/support/KafkaHeaders.java

```diff
@@ -32,4 +33,8 @@
 
 	public static final String OFFSET = PREFIX + "_offset";
 
+	public static final String NEXT_OFFSET = PREFIX + "nextOffset";
+
+	public static final String ACKNOWLEDGMENT = PREFIX + "acknowledgment";
```

Comment:
Yes, we'll make that a separate fix so that we can have a 1.0.x with support for this.